ArduCopter/ArduCopter.cpp

```diff
@@ -202,6 +203,14 @@ void Copter::barometer_accumulate(void)
     barometer.accumulate();
 }
 
+/*
+  adjust baro for changing air pressure
+ */
+void Copter::barometer_adjust(void)
+{
+    barometer.update_alt_target(pos_control.get_alt_target()/100.f);
```

Comment:
My intention is to run at 1Hz. Both GPS and baro run much faster and would require local prescaling. Move anyway?